### PR TITLE
Feature/refactor css

### PR DESCRIPTION
**Pull request by danielsolomon332**
** What code did you implement/add?
- Improve CSS layout. Added hover for info card

** Did you refactor anything?
- Typos is scripts.js

** What resources did you use?
- None

** Are there any issues that came up?
- No

** Is there anything collaborators should know?
- No

** Do you have any questions?
- No

### CODE REVIEW

**Review by Romeslayer — APPROVED** [review]
The webpage looks a lot cleaner.

File: src/css/styles.css [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+}
+
+.user-container {
+  width: 50px;
+  height: 50px;
+  background-color: blue;
+}
+
+.user-container:hover .info-card {
+  display: block;
```
Comment (Prkr93) [inline src/css/styles.css:40]: I'm still lovin' this.

File: dist/index.html [hunk 2]
```diff
@@ -7,17 +7,15 @@
...
       <h2>FitLit</h2>
-      <section>
-        <h1 id="userName">Welcome User</h1>
-        <p id="stepGoal">Your step goal: / Average: </p>
-        <img id="userLogo" />
+      <h1 id="userName">Welcome User</h1>
+      <p id="stepGoal">Your step goal: / Average: </p>
+      <section class="user-container"><img id="userLogo" />
+        <section class="info-card hidden" id="infoCard">
+        </section>
```
Comment (Prkr93) [inline dist/index.html:14]: Might just want to put the closing tag on the same line.

**Review by Prkr93 — APPROVED** [review]
Looks good!

**Pull request merged**